main/settings/src/main/scala/sbt/Reference.scala

```diff
@@ -78,7 +78,7 @@ object Reference {
       case LocalRootProject    => "{<this>}<root>"
       case LocalProject(id)    => "{<this>}" + id
       case RootProject(uri)    => "{" + uri + " }<root>"
-      case ProjectRef(uri, id) => "{" + uri + "}" + id
+      case ProjectRef(uri, id) => s"""ProjectRef(uri("$uri"),"$id")"""
```

Comment:
I don't think it's a common scenario to copy-paste `ProjectRef` from the shell to the DSL. I wouldn't call this consistent -- you are basically outputting scala code to the shell for something that is >95% only meant to be read by the user.
  
Also, no one would write the `uri` completely with `file`, you would probably just refer to the project definition.
  
I am not convinced by this, honestly. I think we should find a better way to show project refs, at least for task and settings references. Remember that this output is shown for every `update`.